File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.swift
// T: Number of Test cases
let T: Int = Int(readLine() ?? "1") ?? 1

for i in 1...T {
    // inputArray: Array<Int> to be used for Addition
    let inputArray: [Int] = readLine()?.split(separator: " ").compactMap{Int($0)} ?? [1, 1]
    
    // 0 < A, B < 10
    let A: Int = inputArray[0]
    let B: Int = inputArray[1]
    
    print("Case #\(i): \(A) + \(B) = \(A+B)")
}

File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.py
# T: Number of Test cases
T = int(input())

for i in range(1, T+1) :
    # A, B : Numbers to be used for Addition
    A, B = map(int, input().split())
    
    print(f"Case #{i}: {A} + {B} = {A+B}")